Code-the-Dream-School/dd-prac-team4-front
# Ps 95 show the results on a checkout complete page

**Pull Request Description**

**Changes Made:**

I have implemented and made changes to several React components to enhance the checkout process. These changes include handling Stripe payment status, loading order details, and displaying relevant information to the user.

**Testing Steps:**

1. Install dependencies:

   
`   npm install`
  

2. Run application:

 Front-end: `  npm start`
 Back-end: `npm run dev`

3. Open the application in your web browser.

4. Navigate to the checkout page and complete a payment transaction.

[http://localhost:3000/checkout](http://localhost:3000/checkout)

5. Observe the following:

   - Ensure that the payment status is displayed correctly.
   - Confirm that the order details are loaded and displayed when payment is successful.
   - Check for error messages in case of payment failure.

7. Verify that the "Return to Marketplace" button works correctly and redirects to the desired page.

<img width="1439" alt="Screenshot 2023-09-05 at 12 31 16 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/2952900/f24d8d88-da37-49c2-aa2a-01eabd51f1e5">
